src/cli/bootstrap/deploy-contract.ts

```diff
@@ -38,4 +42,24 @@ export default async function deployContract(
 
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     await contract.deployContract(Name.from(account) as any, wasmFile, abiFile, signer, options);
```

Comment:
this will still deploy the contract. if options.actionData is set it should not deploy